GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -286,12 +286,12 @@ - (void)setUp {
 #elif TARGET_OS_OSX
   _presentingWindow = OCMStrictClassMock([NSWindow class]);
 #endif // TARGET_OS_IOS || TARGET_OS_MACCATALYST
-  _authState = OCMStrictClassMock([OIDAuthState class]);
+  _authState = OCMClassMock([OIDAuthState class]);
```

Comment:
Please add a link to the issue describing the problem we are addressing here.

Reply:
Done.